arangod/Cluster/ClusterMethods.h

```diff
@@ -51,16 +51,22 @@ class ClusterTraverserCache;
 
 namespace velocypack {
 class Builder;
+class HashedStringRef;
 }
 
 namespace traverser {
 struct TraverserOptions;
 }
 
-struct ClusterCommResult;
 class ClusterFeature;
 struct OperationOptions;
 
+////////////////////////////////////////////////////////////////////////////////
+/// @brief creates a copy of all HTTP headers to forward
+////////////////////////////////////////////////////////////////////////////////
+
+network::Headers getForwardableRequestHeaders(GeneralRequest*);
+
```

Comment:
I don't think we should add this back. This is from 3.6 and should be unneeded now.
```suggestion
```